src/picongpu/include/plugins/output/header/MessageHeader.hpp

```diff
@@ -24,7 +24,7 @@
 #include "simulation_defines.hpp"
 #include "dimensions/DataSpace.hpp"
 #include "iostream"
-#include "stdlib.h"
+#include "cstdlib"
```

Comment:
Ouch! How could this slip through? Please always include system headers with <> style